tests/ui/attributes/unix_sigpipe/unix_sigpipe-inherit.rs

```diff
@@ -1,14 +1,29 @@
+//@ ignore-cross-compile because aux-bin does not yet support it
```

Comment:
Could you open a GH issue for that if you think it'd be convenient to support this?

Reply:
Done with https://github.com/rust-lang/rust/issues/124361